controllers/nodefeaturediscovery_resources.go

```diff
@@ -32,9 +32,13 @@ import (
 	"k8s.io/kubectl/pkg/scheme"
 )
 
+// assetsFromFile is a list where each item in the list contains the
+// contents of a given file as a list of bytes
```

Comment:
A bit cryptic way of explaining a simple thing 😸 

```suggestion
// assetsFromFile is the content of an asset file as raw data
```